### ISSUE TITLE
Missing semicolon in .css ignored

### ISSUE THREAD

**Issue by rdeltour** [w3c/epubcheck#243]
_From [jens.tro...@gmail.com](https://code.google.com/u/100817963473914031312/) on January 24, 2013 12:52:13_

PROBLEM: I forgot to terminate a rule in my stylesheet.css with a semicolon, which caused a lot of rendering issues on the Nook. Epubcheck did not catch the missing semicolon, which would have been nice.

METHOD: Something like this:

```
table {
  width: 100%
  height: 100%;
}
```

RESULT: No warning issued.  Rendering a mess.

EXPECTED: Warning about missing semicolon.

PLATFORM: Epubcheck 3.0 on Mac 10.6.8

```
java version "1.6.0_37"
Java(TM) SE Runtime Environment (build 1.6.0_37-b06-434-10M3909)
Java HotSpot(TM) Client VM (build 20.12-b01-434, mixed mode)
```

ADDITIONAL INFO: n/a

_Original issue: http://code.google.com/p/epubcheck/issues/detail?id=243_

**Comment by tofi86** [comment]
Confirmed. Still happens with 4.0.2 and current dev version. The following CSS snippet does not throw an error:

```css
table {
  width: 100%
  height: 100%;
}
```

result:
```
Validating using EPUB version 3.0.1 rules.
No errors or warnings detected.
epubcheck completed
```

**Comment by rdeltour** [comment]
I'm not sure that missing semicolons are strictly a syntax error, as in this case it makes the line par of the previous declaration.

In any case, closing this as duplicate of the meta-issue #981
